tests/complete_item_spec.lua

```diff
@@ -0,0 +1,58 @@
+local helpers = require('tests.helpers')
+
+describe("NotesCompleteItem", function()
+  local today
+  local tempfile_path
+
+  before_each(function()
+    helpers.setup_test_env()
+    helpers.clear_buffer()
+    -- Ensure plugin is loaded
+    vim.cmd('runtime! plugin/notes.vim')
```

Comment:
Is loading the plugin here again really needed? It is also loaded in minimal_init. Check which one is actually working.